gradle.properties

```diff
@@ -0,0 +1,4 @@
+org.gradle.caching=true
+group=io.ballerinax
+version=7.0.0
+ballerinaLangVersion=2201.0.3
```

Comment:
As discussed offline, lets use 2201.1.0 instead of 2201.0.3

Reply:
Ack fixed in fc08a84ae3b62f98c8863f7b4b430da95533b776